api/src/utils/user-utils.js

```diff
@@ -53,9 +60,17 @@ const filterViewableFields = (currentUser, member) => {
   return filteredMember;
 };
 
+const validateField = (field, value) => {
```

Comment:
For this function lets consider decoupling it from our global state. I would be more modular to pass in the validation fields along with our other arguments.